env0/data_environment.go

```diff
@@ -127,6 +148,30 @@ func dataEnvironmentRead(ctx context.Context, d *schema.ResourceData, meta inter
 		return diag.FromErr(err)
 	}
 
+	subEnvironments := []interface{}{}
+
+	if environment.LatestDeploymentLog.WorkflowFile != nil {
+		for alias, subenv := range environment.LatestDeploymentLog.WorkflowFile.Environments {
+			subEnvironment := map[string]interface{}{
+				"id":    subenv.EnvironmentId,
+				"alias": alias,
+			}
+			subEnvironments = append(subEnvironments, subEnvironment)
+		}
+	}
+
+	slices.SortFunc(subEnvironments, func(a, b interface{}) int {
```

Comment:
the order in a map can be inconsistent...
Added a sort to the slice to be sure order is always the same.

Reply:
this is mostly for tests... but who knows... maybe this inconsistency can also cause real issues in the future...